labels.md

```diff
@@ -0,0 +1,219 @@
+component/test                  #6494f4     C: Test                             #d1fad7
+component/tikv                  #8bef73     C: Tikv                             #d1fad7
+component/tools                 #cbf232     C: Tools                            #d1fad7
+component/transaction-need-LGT  #83e2d1     C: Transaction need LGT             #d1fad7
+component/usability             #0ad8bd     C: Usability                        #d1fad7
+component/util                  #47e0b0     C: Util                             #d1fad7
+contribution                    #b0ff5b     T: Contributor ⭐️                    #1d76db
+errmsg                          #a7c938     T: Error message                    #1d76db
+for new contributors            #c2e0c6     D: Mentor                           #0e8a16
+help wanted                     #159818     S: HelpWanted                       #e6e6e6
+needs-cherry-pick-2.1           #000000     T: CherryPick-2.1                   #1d76db
+needs-cherry-pick-3.0           #000000     T: CherryPick-3.0                   #1d76db
+priority/P1                     #b60205     P: Low                              #eb6420
+priority/P2                     #FF7F50     D: Medium                           #0e8a16
+priority/non-release-blocker    #0052cc     P: High                             #eb6420
+priority/release-blocker        #b60205     P: Blocker                          #eb6420
+proposal                        #eb6420     T: Proposal                         #1d76db
+question                        #fbca04     T: Question                         #1d76db
+release-note                    #1d76db     * <T: CHANGELOG>                    
+require-LGT3                    #d93f0b     * <S: Waiting on review>            
+status/DNM                      #b60205     S: DNM                              #e6e6e6
+status/LGT1                     #d4c5f9     * <S: Waiting on review>            
+status/LGT2                     #5319e7     * <S: Waiting on review>            
+status/LGT3                     #330099     * <S: Waiting on review>            
+status/PTAL                     #2cbe4e     S: Waiting on review                #e6e6e6
+status/ReqChange                #b60205     S: Waiting on author                #e6e6e6
+status/TODO                     #207de5     *                                   
+status/WIP                      #fbca04     S: WIP                              #e6e6e6
+status/all tests passed         #2cbe4e     * <S: Bot Merge>                    
+status/can merge                #2cbe4e     S: Bot Merge                        #e6e6e6
+status/future                   #fbca04     ?                                   
+type/1.0 cherry-pick            #c2e0c6     *                                   
+type/2.0 cherry-pick            #99d0ef     *                                   
+type/2.1 cherry-pick            #9bfff5     * <T: CherryPick-2.1>               
```

Comment:
this label means the PR is a cherry-pick on the 2.1 release-branch.

Reply:
What is this used for? I.e., why does it matter if a PR has been cherry-picked?

Reply:
ping @zz-jason question ^